test/test_avl_1.c

```diff
@@ -35,17 +37,17 @@ int main(int argc, char **argv)
 	int len = sizeof(arr) / sizeof(arr[0]);
 
 	for (; count < len; count++) {
-		if (insert_avl_node(head, arr[count]) == FALSE) {
+		if (insert_avl_node(head, arr[count],20) == FALSE) {
```

Comment:
space after comma... same for line 48 and anywhere else.

Reply:
Did you notice this comment?

Reply:
solve